edx_proctoring/api.py

```diff
@@ -1497,14 +1497,11 @@ def _get_timed_exam_view(exam, context, exam_id, user_id, course_id):
         progress_page_url = ''
         try:
             progress_page_url = reverse(
-                'courseware.views.progress',
+                'courseware.views.views.progress',
                 args=[course_id]
             )
         except NoReverseMatch:
```

Comment:
Do we need to have except here as well. 

I think, this is called by [LMS progress view](https://github.com/edx/edx-platform/blob/5346ee25b8eed7d13eec9dc451f22e442909c9b3/lms/djangoapps/courseware/module_render.py#L272) which is passing valid course id, so if the view is passed correctly in the reverse method, I don't think, we need exception block here.

@hasnain-naveed  was there any specific reason this was added ?